giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/service/properties/Properties.java

```diff
@@ -49,4 +52,6 @@
     public final static String GILES_TMP_FOLDER = "giles_files_tmp_dir";
 
     public final static String FREDDIE_HOST = "freddie_host";
+    
+    public final static String MITREID_CONNECT_ACCESSTOKEN = "mitreidconnect_accessToken";
```

Comment:
this doesn't belong here, but you wont' need it anyways with the above mentioned changes.